.github/workflows/test.yml

```diff
@@ -329,3 +320,13 @@ jobs:
             --recursive
             --config=${{ github.workspace }}/examples/.tflint.hcl
           fail_on_error: true
+
+  release:
+    needs: [test, lint, mock, k3s, localstack, docker]
+    if: github.event_name == 'push'
+    uses: cloudposse/.github/.github/workflows/shared-go-auto-release.yml@main
+    with:
+      publish: false
+      format: binary
+    secrets: inherit
```

Comment:
i don't agree that `release` should be in a workflow called `test.yml` with `name: Tests`

but it was already here to begin with so i won't require it to move out

Reply:
@goruha I think @kevcube brings up a good point naming wise. I would not think any release happens from a workflow file called test.